WooCommerce/Classes/ViewRelated/ReusableViews/TextFieldTableViewCell.swift

```diff
@@ -4,14 +4,29 @@ import UIKit
 ///
 final class TextFieldTableViewCell: UITableViewCell {
     struct ViewModel {
+        internal init(text: String? = nil, placeholder: String? = nil, onTextChange: ((String?) -> Void)? = nil,
```

Comment:
nit: `internal` is default access level, so we can remove it.
```suggestion
        init(text: String? = nil, placeholder: String? = nil, onTextChange: ((String?) -> Void)? = nil,
```